services/supervisord/supervisord.go

```diff
@@ -579,6 +579,9 @@ command =
 		--search.disableCache={{ .VMDBCacheDisable }}
 		--search.maxQueryLen=72KB
 		--search.latencyOffset=5s
+		--search.maxConcurrentRequests=32
+		--search.maxQueueDuration=20s
```

Comment:
According to the description:

https://hub.docker.com/r/victoriametrics/victoria-metrics/

> The maximum time the request waits for execution when -search.maxConcurrentRequests limit is reached (default 10s)

if we 33 concurrent "slow" requests (may run maxQueryDuration=30s) run in concurrently so the 33rd query needs to wait 30 sec before accept to execution, right?

Reply:
Good point. WIll set maxQueueDuration bigger than maxQueryDuration.

Reply:
@JiriCtvrtka 
Could you clarify that I got it right?
maxQueryDuration -> lifetime of a general query (e.g. show me a metric for 30 days)
maxQueueDuration -> lifetime of a request as a result of splitting a general query by services, types .... (e.g. 80 requests)

So if we have eight core CPU and spends 1 sec for  each request than a query (80 requests) would be processed by 20 seconds.

The question is next - maxQueueDuration is been started counting since a query is generated (spitted to 80 requests) or when a request tried to be executed and stacked due CPU core occupation?
If the first statement is correct than query maxQueryDuration duration is useless because a whole query would be break after maxQueueDuration time. 

So most probably the second statement is correct. In such case it's not reasonable increasing maxConcurrentRequests to 32.

My suggestion is to just double default values for all involved parameters.
In such case maxQueryDuration will be equal to Grafana Query timeout.

Reply:
I checked it and everytime when I changed "refresh interval" then was created around 77 requests (query, query_range) in PMM demo. Not sure if set 16 (double default value) will be enough. Lets wait also for Nurlan opinion.